src/Series.cpp

```diff
@@ -350,6 +350,16 @@ Series::setName(std::string const& n)
     return *this;
 }
 
+void Series::setFlush(FlushType flushType)
```

Comment:
Maybe we want to include this in the `Series::Series()` constructor with `DIRECT` as default to make the behaviour a little more obvious?

Reply:
I thought about this and find it has enough arguments.
If one wants to activate the "advanced" usage, I like that it is an explicit step.